packages/components/psammead-program-card/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@bbc/psammead-program-card",
```

Comment:
I'm wondering if it would be easier for us to create a `psammead-radio-schedule` package that contains the program card, the timestamp, and the wrapping components to begin with, and then split things out into separate packages if it becomes apparent later that we can make them generic and reuse them across new components? Probably something to discuss with the team.

Reply:
After discussion, we've decided to have a `psammead-radio-schedule` package that exports a `program-card` component among others.